config/config.go

```diff
@@ -165,7 +157,7 @@ func NewStoresConfig() *StoresConfig {
 func (f *configFile) load(bytes []byte) error {
 	err := yaml.Unmarshal(bytes, f)
 	if err != nil {
-		return fmt.Errorf("Could not unmarshal config file: %s", err)
+		return fmt.Errorf("could not unmarshal config file: %s", err)
```

Comment:
Would you mind moving this change to another PR so we can merge the rest *now* instead of somewhen in the future? :)